### ISSUE TITLE
Scanning images from Google Authenticator export mashes issuer and name

### ISSUE THREAD

**Issue by robertdahlem** [beemdevelopment/Aegis#1133]
### Version

v2.1.3

### Source

Google Play

### Vault encryption

No

### Device

Motorola Moto G60s

### Android version

Android 12

### Steps to reproduce

- Google Authenticator v6.0 > Transfer accounts > Export accounts > Select all accounts > make a hardcopy of displayed QR code
- Aegis v2.1.3 > "plus" button > Scan image > select hardcopy from Photos

### What do you expect to happen?

Aegis imports all accounts with name and issuer separated.

### What happens instead?

Aegis imports accounts with name as "Issuer:name" and issuer as "issuer".

For example:
In Google Authenticator I have Issuer="Amazon" and Name="me@example.com"
This becomes Name="Amazon:me@example.com" and Issuer="Amazon" in Aegis.

**Comment by michaelschattgen** [comment]
Thanks for your report. I've tried reproducing this but it works on my phone and on my emulator. What is the format exactly that you see in the Google Authenticator app? I just made a few entries in Google Authenticator as "Amazon:Michael", 1 created manually within the app and 1 added through a QR code with Amazon as issuer and Michael as name, both work as intended in Aegis.

I can only see it failing if your format is any different than what I stated above, if not we would have to dig into this further.
